### PR TITLE
add value to analysisType

### PR DESCRIPTION
**Pull request by avanlinden**
add Mendelian randomization analysis value

### CODE REVIEW

File: terms/experimentalData/diagnosis.json [hunk 1]
```diff
@@ -187,6 +187,36 @@
             "const": "traumatic brain injury",
             "description": "A form of acquired brain injury which occurs when a sudden trauma causes damage to the brain.",
             "source": "http://purl.obolibrary.org/obo/OAE_0002392"
+        },
+        {
+            "const": "lewy body dementia",
+            "description": "A progressive form of dementia characterized by the presence of protein deposits called Lewy bodies in the midbrain and cerebral cortex, and loss of cholinergic and dopaminergic neurons. The signs and symptoms overlap with Alzheimer's and Parkinson's disease.",
+            "source": "http://purl.obolibrary.org/obo/MONDO_0007488"
+        },
```
Comment (Aryllen) [inline terms/experimentalData/diagnosis.json:195]: We have "Lewy body dementia".

**Review by Aryllen — CHANGES_REQUESTED** [review]
Just one requested change below. Otherwise, looks good.

File: terms/experimentalData/diagnosis.json (hunk 1 above)
Comment (avanlinden) [reply terms/experimentalData/diagnosis.json]: Checked w/ Mette and removed the proposed "Lewy body dementia"; we can use "Lewy body disease".

**Review by Aryllen — APPROVED** [review]
LGTM!

**Pull request merged**